.gitignore

```diff
@@ -141,8 +141,10 @@ Thumbs.db
 
 # ignore all build directories
 
-*build*
+*build-*
 .qmake.stash
 *Makefile*
 .qtc_clangd
 mocinclude.opt
+build/
```

Comment:
FYI @jared-2016 I snuck this change in. I was getting warnings that files were being ignored that we probably didn't intend, like CppSamples/DisplayInformation/BuildLegend and CppSamples/Maps/BrowseBuildingFloors.